themes/default/content/templates/serverless-application/gcp/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Google Cloud Serverless Application"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The Google Cloud Serverless Template makes it easy to deploy a serverless application on GCP with Pulumi, Google Cloud Functions, and Google Cloud Storage.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a serverless application on Google Cloud with Pulumi, Google Cloud Functions, and Google Cloud Storage.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: serverless-gcp
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Google Cloud Platform
+  slug: gcp
+
+meta_image: meta.png
+
+---
+
+The Serverless Application template creates an infrastructure as code project in your favorite language that deploys a serverless application to Google Cloud Platform with Pulumi. It deploys a [Google Cloud Storage bucket]({{< relref "/registry/packages/gcp/api-docs/storage/bucket" >}}) configured for static website hosting and another bucket to host the source code for a [Cloud Function]({{< relref "/registry/packages/gcp/api-docs/cloudfunctions/function" >}}) written in the same language as the template. It uses a [synced folder]({{< relref "/registry/packages/synced-folder/api-docs/googlecloudfolder" >}}) to manage website files. The template ships with placeholder content to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
We don't mention this implementation detail in other templates, so unless we think it's relevant here, I'd just as soon leave it out for consistency:

```suggestion
The Serverless Application template creates an infrastructure as code project in your favorite language that deploys a serverless application to Google Cloud Platform with Pulumi. It deploys a [Google Cloud Storage bucket]({{< relref "/registry/packages/gcp/api-docs/storage/bucket" >}}) configured for static website hosting and another bucket to host the source code for a [Cloud Function]({{< relref "/registry/packages/gcp/api-docs/cloudfunctions/function" >}}) written in the same language as the template. The template ships with placeholder content to give you a working project out of the box that you can customize easily and extend to suit your needs.
```